docs/products/kafka/howto/tiered-storage-overview.rst

```diff
@@ -0,0 +1,66 @@
+Tiered storage overview in Aiven Console
+========================================
+
+Aiven for Apache Kafka® offers a comprehensive overview of tiered storage, allowing you to understand its usage and make informed decisions. This overview provides insights into various aspects of tiered storage, including billing, settings, and storage details.
+
+.. important:: 
+    
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
+
+
+Access tiered storage overview
+--------------------------------
+
+1. In the Aiven Console, choose your project and select your Aiven for Apache Kafka service.
+2. From the left sidebar, select **Tiered Storage**. This action will display an overview of tiered storage and its associated details.
+
+
+Key insights of tiered storage
+------------------------------
+
+Get a quick snapshot of the essential metrics and details related to tiered storage:
+
+- **Current billing expenses in USD**: Stay informed about your current tiered storage expenses.
+- **Forecasted month cost in USD**: Estimate your upcoming monthly costs based on current usage.
+- **Data tiered in bytes**: View the volume of data that has been tiered.
+- **Storage overview**: Understand the specifics of the storage mediums in use, including details about the used object storage and SSD storage.
+
+
+Current tiered storage configurations
+---------------------------------------------
+
+View of the current configurations for tiered storage:
+
+- **Local Cache**: Shows the current cache configuration.
+- **Default Local Retention Time (ms)**: Shows the current local data retention set in milliseconds.
```

Comment:
Wondering if these should mention the actual setting name? 🤔

Reply:
Yes, those are the actual settings names. I'm just thinking that if for example you use the CLI, you would use the real setting names so it could be reassuring that we are in fact talking about the exact same service level settings. 

I think we should just use the real setting name in console as well to avoid confusion.